### PR TITLE
Update lists.py with trivia questions created for Action #28

### LINKED ISSUE #28: Write questions and answers for the trivia command (Max)
Write at least 5-10 questions for use with the trivia command in User Story #9. Questions should have answers that are one or two words. Questions/answers can be initially written as plaintext, then upon merging of the pull request associated with Task/Action #25, converted to code and a pull request filed.

Completion Criteria: At least 5 trivia questions (more would be great :smile:) with one- or two-word answers are prepared, coded in the format implemented in #25, and a pull request with that code is made.

Due: Thursday, March 25, or 72 hours after the pull request for #25 is merged, whichever is later

### PR DESCRIPTION
**Pull request by mdlubbers167**
I added the trivia questions to the code as required by Action #28

### CODE REVIEW

File: utils/lists.py [hunk 1]
```diff
@@ -10,21 +10,13 @@
...
-    QuestionAnswer("What plant is known to help heal a sunburn?", "Aloe"),
-    QuestionAnswer("How many planets are in our solar system? (Including Pluto!)", "9"),    
-
+    QuestionAnswer("What is your body’s largest organ?", "Skin"),
+    QuestionAnswer("What year did the Titanic movie come out?", "1997"),
+    QuestionAnswer("Who played Neo in The Matrix?", "Keanu Reeves"),
+    QuestionAnswer("What year was the first IPhone released?", "2007"),
+    QuestionAnswer("Which bone are babies born without?", "Knee Cap"),
+    QuestionAnswer("What is Hawkeye's real Name?", "Clint Barton"),
+    QuestionAnswer("Mexico’s Dia de los Muertos, means what in English?", "Day of the Dead"),
```
Comment (jcgoble3) [inline utils/lists.py]: You're working off an old version of `main`. Please pull the latest commits to `main` and rebase your work onto the top of the current `main` branch. You will have to fix a conflict in the process, then force-push.

To start:
```sh
git checkout main
git pull
git checkout max
git rebase main
```

You will then have a merge conflict to fix. This guide might be helpful: http://gitforteams.com/resources/rebasing.html Make sure to keep all of the trivia questions from both changes (order doesn't matter), remove the conflict markers, then `git add utils/lists.py` and `git rebase --continue`.

Then your local branch and the remote branch will have diverged, so run `git push --force-with-lease` to do a forced update to this pull request.

**Review by jcgoble3 — CHANGES_REQUESTED** [review]
Questions look good, but the latest changes need to be integrated.

File: utils/lists.py (hunk 1 above)
Comment (sethsarchet) [reply utils/lists.py]: This is looking good, but may want to hold off on merging this for a little bit. The format of how we will be reading questions in will now be from a text file. So when that gets merged, would probably be better to put these questions in the trivia_questions_answer.txt file.

Comment (jcgoble3) [reply utils/lists.py]: Fixes look good, thanks! I'll leave the conflict fix up to Seth as I think it will be simpler that way.

**Review by jcgoble3 — APPROVED** [review]

**Comment by dward118** [comment]
We could have split the trivia questions into multiple categories and could have had the user pick between the options. Like having sports related trivia questions under a certain section for example. It could give the user a little more of a preference to what questions they would try to figure out.

**Pull request merged**